src/server/cmd/pachd/main.go

```diff
@@ -37,6 +38,8 @@ func main() {
 	logMode := func(mode string) {
 		log.Info(ctx, "pachd: starting", zap.String("mode", mode))
 	}
+	go proc.MonitorSelf(ctx)
+
```

Comment:
This is where I think we should create the logging MetricSpace and attach it to the context.  I actually think we should call `zap.NewProduction` here too, but that's a discussion for another day.